xbps-src

```diff
@@ -434,6 +434,13 @@ if [ -d "$XBPS_MASTERDIR" -a ! -w "$XBPS_MASTERDIR" ]; then
     exit 1
 fi
 
+# Forbid root unless XBPS_ALLOW_CHROOT_BREAKOUT is set
+# (for travis CI).
+if [ -z "$IN_CHROOT" -a "$UID" -eq 0 -a -z "$XBPS_ALLOW_CHROOT_BREAKOUT" ]; then
```

Comment:
1. I propose  `"$(id -u)" = "0" ` instead of `"$UID" -eq 0`
2. why not around L368 (check argnum), where other early exit is handled?

Reply:
not forking > POSIX compliance in explicit bash script.

Reply:
May be better to put between reading config in 422 and directory creation in line 426. Documentation should mention this too.

Reply:
@Piraty $UID is simpler and faster

@Chocimier indeed, good suggestion